server/Player.js

```diff
@@ -44,9 +49,35 @@ export default class Player {
     );
   }
 
+  get #invisibleGridWithTeromino() {
```

Comment:
```suggestion
  get #invisibleGridWithTetromino() {
```
I think it should be a method instead of a property too, but that will surely change if we have the other method `arrayWithShadowTetromino` anyways

Reply:
hmm I'm not sure I understand but I'm guessing its in a future pr, I did fix the typo for now.

Reply:
Wdym? Why it shouldn't be a property? I think it's because it does calculations instead of just `getting` directly

Reply:
I did not put much thought into it, I just used `  get #gridWithTetromino() {` to work from.